Volume/View Models/TrendingViewModel.swift

```diff
@@ -65,24 +65,12 @@ extension TrendingView {
         }
         
         func fetchFlyers() async {
-            // TODO: Change query once backend implements trending
-            guard let url = URL(string: "\(Secrets.cboardEndpoint)/flyers/trending/") else { return }
-            
-            URLSession.shared.dataTaskPublisher(for: url)
-                .subscribe(on: DispatchQueue.global(qos: .background))
-                .receive(on: DispatchQueue.main)
-                .tryMap { data, response in
-                    guard let response = response as? HTTPURLResponse,
-                          response.statusCode >= 200 && response.statusCode < 300 else {
-                        throw URLError(.badServerResponse)
-                    }
-                    return data
-                }
-                .decode(type: [Flyer].self, decoder: JSONDecoder.flyersDecoder)
-                .sink { completion in
-                    print("Fetching trending flyers: \(completion)")
-                } receiveValue: { [weak self] flyers in
-                    self?.flyers = flyers
+            Network.shared.publisher(for: GetTrendingFlyersQuery(limit: 2))
```

Comment:
why is the limit 2?

Reply:
The trending page only displays 2 flyers.